lib/main.js

```diff
@@ -0,0 +1,41 @@
+import { getFullVersionFromSnForge, versionWithPrefix } from "./versions";
+import { downloadSnForge } from "./download";
+import { getOsTriplet } from "./platform";
+import path from "path";
+import * as core from "@actions/core";
+import * as tc from "@actions/tool-cache";
+
+export default async function main() {
+  try {
+    const snforgeVersionInput = core.getInput("version");
+
+    const { repo: SnForgeRepo, version: SnForgeVersion } = {
```

Comment:
let's also rename all occurences of snforge here to starknet-foundry or snfoundry
@MaksymilianDemitraszek not sure which one should be _the default_

Reply:
can I change into SnFoundry?

Reply:
snfoundry imo as it is more succinct

Reply:
@MaksymilianDemitraszek should this be camel case or not? I like `snfoundry`, `snforge` and `sncast` (in log messages etc - not necessarily in function names, since it's js) over `SnFoundry` etc, but not sure that really matters